tests/mir-opt/copy-prop/mutate_through_pointer.rs

```diff
@@ -1,3 +1,13 @@
+// This attempts to mutate a via a pointer derived from addr_of!(a). That is UB
```

Comment:
"to mutate a ? via a pointer"?

Reply:
We mean the variable called `a`. I tried adding backticks to make it clear what is Rust and what is English.

Reply:
Ouch, lol... yes thanks that should help.